docs/operations/export_api.md

```diff
@@ -200,6 +200,55 @@ status:
   phase: Ready
   serviceName: virt-export-example-export
 ```
+
+#### Manifests
+The VirtualMachine manifests can be retrieved by accessing the `manifests` in the VirtualMachineExport status. The `all` type will return the VirtualMachine manifest, any DataVolumes, and a configMap that contains the public CA certificate of the Ingress/Route of the external URL, or the CA of the export server of the internal URL. The `auth-header-secret` will be a secret that contains a Containerized Data Import (CDI) compatible header. This header contains a text version of the export token.
+
+Both internal and external links will contain a `manifests` field. If there are no external links, then there will not be any external manifests either.
+
```

Comment:
Should we also add something like: 
VirtualMachine manifest will be present only if VirtualMachineExport was created for VirtualMachine or VirtualMachineSnapshot. Export of PersistentVolumeClaim will not generate a VirtualMachine manifest.

Reply:
Done